.github/workflows/ci.yml

```diff
@@ -140,8 +153,11 @@ jobs:
             -DCMAKE_INSTALL_PREFIX=../install \
             -DCMAKE_PREFIX_PATH="$JlCxx_DIR;$OpenFHE_DIR" \
             -DJulia_PREFIX="$(dirname $JULIA_BINDIR)" \
-            -G "Unix Makefiles"
-          make -j 2
+            -G "Unix Makefiles" \
+            -LDFLAGS="-lc++abi" \
```

Comment:
`LDFLAGS` is not a CMake flag. What you need to do is set it above, in the invocation of `cmake` itself, e.g.,
```shell
LDFLAGS="-lc++abi" cmake ..
```
or export it before the `cmake` command, e.g.
```shell
export LDFLAGS="-lc++abi"
cmake ..
```

Reply:
Vielen Dank!